fetch.bs

```diff
@@ -5887,7 +5887,10 @@ object), initially null.
 
    <dt>{{RequestInit/credentials}}
    <dd>A string indicating whether credentials will be sent with the request always, never, or only
-   when sent to a same-origin URL. Sets <var>request</var>'s {{Request/credentials}}.
+   when sent to a same-origin URL — as well as whether any credentials sent back in the response
+   will be used always, never, or only when received from a same-origin URL. Sets
+   <var>request</var>'s {{Request/credentials}}. If {{RequestInit/credentials}} is unspecified, the
+   <a>credentials mode</a> for <var>request</var> defaults to "<code>same-origin</code>".
```

Comment:
It's only the default if _input_ is a string (otherwise it would end up overriding the credentials mode).

Reply:
> It's only the default if _input_ is a string (otherwise it would end up overriding the credentials mode).

OK, pushed another commit which corrects the text to state that